.pre-commit-config.yaml

```diff
@@ -17,25 +17,25 @@ repos:
         - flake8-unused-arguments==0.0.6
         - flake8-use-fstring==1.3.0
 - repo: https://github.com/psf/black
-  rev: 22.3.0
+  rev: ae2c0758c9e61a385df9700dc9c231bf54887041
```

Comment:
Isn't it better to add the version after as a comment for clarity ? Like `rev: ae2c0758c9e61a385df9700dc9c231bf54887041 # 22.3.0`